modules/util/portal-tools-soy-builder/src/main/java/com/liferay/portal/tools/soy/builder/maven/BaseSoyJsMojo.java

```diff
@@ -34,7 +39,24 @@ public BaseSoyJsMojo() {
 	@Override
 	public void execute() throws MojoExecutionException {
 		try {
-			command.execute();
+			if (_buildContext.isIncremental()) {
+				Scanner scanner = _buildContext.newScanner(_baseDir);
+
+				String[] includes = {"", "**/*.soy"};
```

Comment:
Mojos that extend `BaseSoyJsMojo` process `.soy.js` files, not `.soy` files. Shouldn't we change this?